app/controllers/application_controller.rb

```diff
@@ -1592,22 +1592,23 @@ def find_checked_items(prefix = nil)
   def process_saved_reports(saved_reports, task)
     success_count = 0
     failure_count = 0
-    MiqReportResult.find_all_by_id(saved_reports, :order => "lower(name)").each do |rep|
-      id = rep.id
-      rep_name = rep.name
-      if task == "destroy"
-        audit = {:event => "rep_record_delete", :message => "[#{rep_name}] Record deleted", :target_id => id, :target_class => "MiqReportResult", :userid => current_userid}
-      end
+    MiqReportResult.where(:id => saved_reports).order("lower(name)").each do |rep|
       begin
-        rep.public_send(task.to_sym) if rep.respond_to?(task)    # Run the task
-      rescue StandardError => bang
+        rep.public_send(task) if rep.respond_to?(task) # Run the task
+      rescue StandardError
```

Comment:
No need for the `StandardError` bit either...rescue, by default, rescues StandardErrors